drivers/exec/driver.go

```diff
@@ -181,6 +195,23 @@ func (c *Config) validate() error {
 	return nil
 }
 
+func (c *Config) setDeniedIds() error {
```

Comment:
@tgross calling this out as this is the new single-spot to do the initial parsing of the strings and then those get converted to IDRanges which are stuck on the config.

Code definitely feels cleaner as a result thanks.

The Docker driver had a similar pattern with some config attributes, but I'm not positive there's not a nicer way than having two similar attrs on the struct (see above).

Reply:
> The Docker driver had a similar pattern with some config attributes, but I'm not positive there's not a nicer way than having two similar attrs on the struct (see above).

Yeah that's pretty much it. If you dig a little bit in other config values you'll see that's how we end up having to do pretty much anything that needs special parsing (ex. parsing "3h" as `time.Duration(3 * time.Hour)`)